app/timeline-menu/timeline-menu.ts

```diff
@@ -0,0 +1,63 @@
+let _ = require('lodash');
+
+export enum TimelineMenuRange { LastWeek, LastMonth, LastYear, LastDecade, PreDecade, CustomRange }
```

Comment:
why is this exported?

Reply:
I use the enum in external components such as search component for example